src/components/modal/Modal.js

```diff
@@ -29,10 +29,7 @@ class Modal extends React.Component {
 
     return (
       <div className={styles.modal} onClick={onClose}>
-        <div onClick={(e) => e.preventDefault()} className={styles.innerModal}>
-          <button onClick={onClose} className={styles.closeButton}>
-            x
-          </button>
+        <div onClick={(e) => e.stopPropagation()} className={styles.innerModal}>
```

Comment:
Does this works when clicking outside the Modal? I did not use it because the Modal was not closing then. Just to be aware of this. @crysfel

Reply:
Yes, it works. With `preventDefault` it was closing the modal even when clicking the content. When clicking outside the modal the event propagates and the modal gets closed as expected.